docs/modules/ROOT/pages/governance.adoc

```diff
@@ -197,3 +202,229 @@ pub trait TimelockABI<TState> {
     fn renounceRole(ref self: TState, role: felt252, account: ContractAddress);
 }
 ----
+
+== Votes
+
+The {votes-component} provides a flexible system for tracking voting power and delegation. This system allows users to delegate their voting power to other addresses, enabling more active participation in governance.
+
+NOTE: By default, token balance does not account for voting power. This makes transfers cheaper. The downside is that it requires users to delegate to themselves in order to activate checkpoints and have their voting power tracked.
+
+IMPORTANT: Transfering of voting units must be handled by the implementing contract. In the case of `ERC20` and `ERC721` this is usually done via the hooks. You can check the {votes-usage} section for examples on how to implement this.
+
+=== Key Features
+
+1. *Delegation*: Users can delegate their voting power to any address, including themselves.Vote power can be delegated either by calling
+the {delegate} function directly, or by providing a signature to be used with
+{delegate_by_sig}.
+2. *Historical lookups*: The system keeps track of voting power at different points in time, allowing for accurate voting in proposals that span multiple blocks.
+
+=== Usage
+When integrating the Votes component, the `VotingUnitsTrait` must be implemented to get the voting units for a given account as a function of the implementing contract. For simplicity, this module already provides two implementations for ERC20 and ERC721 tokens, which will work out of the box if the respective components are integrated.Additionally, you must embed the {src-component} to enable delegation by signatures.
```

Comment:
```suggestion
When integrating the Votes component, the `VotingUnitsTrait` must be implemented to get the voting units for a given account as a function of the implementing contract. For simplicity, this module already provides two implementations for ERC20 and ERC721 tokens, which will work out of the box if the respective components are integrated. Additionally, you must embed the {src-component} to enable delegation by signatures.
```

Reply:
We try to split sentences into multiple lines with the main purpose of making them easier to follow here in reviews.

Reply:
> Additionally, you must embed the {src-component} to enable delegation by signatures.

The {src-component} reference doesn't exist. Are we referring to NoncesComponent? If that's the case we should use "implement" instead of "embed" since we implement components and embed embeddable implementations.

Reply:
- Splitting into multilple lines: Good idea, makes it more readbale. But just FYI I don't think we've been very careful with that across the docs when inside the same paragraph.
- Updated the reference to nonce component and SNIP12Metadata